src/apps/project-request/RequestResultMessage.tsx

```diff
@@ -0,0 +1,55 @@
+import * as AlertDialog from '@radix-ui/react-alert-dialog';
+import type { Translations } from 'src/Types';
+import { AnimatedCheck } from '@components/AnimatedIcons';
+
+import './RequestResultMessage.css';
+import { AnimatedFailure } from '@components/AnimatedIcons/AnimatedFailure';
+
+interface RequestResultMessageProps {
+  open: boolean;
+  i18n: Translations;
+  onClose(): void;
+  state: 'success' | 'failure';
+}
+
+export const RequestResultMessage = (props: RequestResultMessageProps) => {
+  const { open } = props;
+  const { t } = props.i18n;
+  return (
+    <AlertDialog.Root open={open}>
+      <AlertDialog.Portal>
+        <AlertDialog.Overlay className='request-result-overlay' />
```

Comment:
Is there a reason you needed to define custom styles for this? There are global dialog styles in `src/themes/default`. Normally, you just would need to use the appropriate classes (`dialog-overlay`, `dialog-content`, `dialog-title`,  `dialog-description`) and the stylesheet should handle the reset. Or am I missing something?

Reply:
Eh. Sorry. Just a copy/paste thing from elsewhere.  I will clean it up.

Reply:
Actually AlertDialog and Dialog are different. This classes do not work for this.